_config.yml

```diff
@@ -0,0 +1,7 @@
+collections:
+  services:
+    output: true
+    permalink: revops/:collection/:name
```

Comment:
Are services a thing under revops? Or is it a general thing you want to introduce?

If e.g. pipedrive or fractional COO are services, I think they should have top-level pages, e.g. deranged.dk/pipedrive and deranged.dk/fractional-coo or just deranged.dk/coo so they are easy to share.

Placing them under /services doesn't add anything imo?

Reply:
Addressed in https://github.com/derangeddk/deranged.dk/commit/74a3976d56193c03078f443fd2f16c56f800b713